modular-docs-manual/files/TEMPLATE_ASSEMBLY.adoc

```diff
@@ -1,52 +1,48 @@
-//  * Don't use any CAPS
+// Ideally, base the name of the file on the title to avoid confusion.
+// Use a consistent system for file names and IDs. For recommendations, see section "Anchor Names and File Names" in the Modular Documentation Reference Guide.
 
-_Use this template to document a user story -- any series of tasks that the user must perform in a specific order to accomplish something._
+This paragraph explains what the user will accomplish by working through the modules in the assembly and sets the context for the user story the assembly is based on. Can include more than one paragraph. Consider using the information from the user story.
 
-This paragraph explains what the user will accomplish by working through the tasks in sequence. It sets context or explains the benefit the user will gain by performing the tasks that comprise the story. May include more than 1 paragraph, and may include concept modules if needed to fully explain the context, benefits, and other things the user must understand to complete the tasks successfully.
-
-[discrete]
+[id='prerequisites-{context}']
 == Prerequisites
 
-* Sentence or a bulleted list of pre-requisites that must be in place or done before the user starts this task.
-
-* Delete section title and bullets if the task has no required pre-requisites.
-
-* Text can be a link to a pre-requisite task that the user must do before starting this task.
+* A bulleted list of conditions that must be satisfied before the user starts following this assembly.
 
-[discrete]
-== Procedure
+* You can also link to other modules or assemblies the user must follow before starting this assembly.
 
-_Can link to or include task modules, depending on the writer's choice based on information needs. If you link to task modules, add each link as 1 step in a numbered list. The order of included task modules or of the linked steps is important to a repeatable successful outcome. Following is an example of linking to task modules in numbered steps._
+* Delete the section title and bullets if the assembly has no prerequisites.
 
-. Link to 1st task to perform.
+_The following include statements pull in the module files that comprise the assembly. Include any combination of concept, procedure, or reference modules required to cover the user story. You can also include other assemblies._
 
-. Format the step line as an unnumbered bullet if the procedure includes only 1 step (exception to numbering the steps).
+include::modules/path-to-a-module-file.adoc[leveloffset=+1]
+// [leveloffset=+1] ensures that when a module starts with a level-1 heading (= Heading), the heading will be interpreted as a level-2 heading (== Heading) in the assembly.
 
-. Link to 3rd task to perform.
+include::modules/path-to-another-module-file.adoc[leveloffset=+1]
 
-. Link to the Nth task to perform.
+include::path-to-an-assembly-file.adoc[leveloffset=+1]
 
-[discrete]
+[id='additional-resources-{context}']
 == Additional Resources
```

Comment:
My only concern here is the placement of the assembly's "Additional Resources". If the last module in the assembly has additional resources, the presentation in the doc will be two Additional Resources sections (the final part of the module, followed immediately by the final part of the assembly). Also, if the assembly has a lot of modules, a reader could easily misinterpret what the assembly's Additional Resources pertains to (since the section would be so far apart from the assembly title and prereqs.

With that said, it is semantically correct to include the "Additional Resources" section just as you've presented it here, so I'm not sure that we should change anything.

Reply:
Agreed, this can be a bit risky. The module template include [discrete] headings for Additional Resources, while the assembly template doesn't -- so the assembly's resources will be displayed as a standard section with a numbered heading. I hope that helps at least a little bit -- the two sections for resources won't look the same.

Some time ago, I played with the idea of including the assembly's additional resources somewhere at the beginning of the assembly instead of at the end -- but that would probably introduce a whole new set of problems :)

Anyway, I think that until we have an alternative solution, we can leave this part of the template as it is.